test/users.spec.js

```diff
@@ -117,4 +121,29 @@ describe('Authentication', () => {
         });
     });
   });
+
+  describe('GET /api/users?token', () => {
+    const baseUrl = '/api/users/verify';
+    it('should return true if a user is verified', done => {
+      chai
+        .request(server)
+        .get(`${baseUrl}?token=${generateToken(tokenPayload)}`)
+        .end((err, res) => {
+          expect(res).to.have.status(200);
+          expect(res.body.verified).to.equal(true);
+          done(err);
+        });
+    });
+
+    it('should check that the token is valid', done => {
```

Comment:
Would also like to see a test for when there's no token passed

Reply:
Has this been implemented?

Reply:
Yes it has been implemented